### PR TITLE
Add 'how to report an issue' flowchart

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- title

### CODE REVIEW

**Comment by commy2** [comment]
Add to https://github.com/acemod/ACE3/blob/master/.github/ISSUE_TEMPLATE/bug_report.md after merge.

**Comment by jonpas** [comment]
I would add them here: https://github.com/acemod/ACE3/tree/master/docs/img/wiki/development

And use underscores instead of spaces for file name.

I would add it under "Steps to reproduce:" as a first line. Just link to it.

**Comment by commy2** [comment]
>Did you follow this flowchart?
>*link*

**Comment by jonpas** [comment]
Far right, it says "Github", needs to be changed to "GitHub". Possibly some other typos.

File: docs/wiki/user/how-to-report-an-issue.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ parent: wiki
 
 ### Before reporting
 
+<a href="{{ site.baseurl }}/img/wiki/user/issue_flowchart.png">
```
Comment (bux) [inline docs/wiki/user/how-to-report-an-issue.md]: Why a link?

Comment (jonpas) [reply docs/wiki/user/how-to-report-an-issue.md]: > Insert a suggestion

;)

**Comment by commy2** [comment]
Well, do you "possibly" have found some other typos or is _company label X_ all?

**Comment by jonpas** [comment]
None that I see, but English is not my first language.

I am slightly bothered by statements that have a question mark at the end though, eg. "I am playing on a premade mission?", should ideally be "Am I playing on a premade mission?".

**Comment by commy2** [comment]
Updated it.

**Comment by jonpas** [comment]
@bux 

image

**Pull request merged**